docs/framework/whats-new/index.md

```diff
@@ -41,9 +41,9 @@ This article does not provide comprehensive information about each new feature a
 
 You can download .NET Framework 4.8  from the following locations:
 
-- [.NET Framework 4.8 Web Installer](https://go.microsoft.com/fwlink/?LinkId=2085155)
+- [.NET Framework 4.8 Web Installer](https://dotnet.microsoft.com/download/dotnet-framework/net48)
```

Comment:
```suggestion
- [.NET Framework 4.8 Web Installer](https://dotnet.microsoft.com/download/dotnet-framework/thank-you/net48-web-installer)
```

Reply:
As stated, I don't want to have deep links.